connector/connect/bin/spark-connect

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Go to the Spark project root directory
+FWDIR="$(cd "`dirname "$0"`"/../../..; pwd)"
+cd "$FWDIR"
+export SPARK_HOME=$FWDIR
+
+# Build the jars needed for spark submit and spark connect
+build/sbt package
```

Comment:
Hm, I guess this can be replaced by `./sbin/start-connect-server.sh --jars `ls connector/connect/server/target/**/spark-connect*SNAPSHOT.jar`?

Reply:
This is more aimed at development. There are quite a few differences:
- This builds the code if we need it
- This does not return after starting the process. It actually logs to the console which makes debugging easier.

Reply:
Should probably put it in `dev` directory if this is for dev ..

Reply:
sorry I misread the path.

Reply:
This scripts ensures that the jars are built up-to-date. So that we can run the client/server to test out the new code quickly. This could be useful for example to test/debug udf etc. This is also why we keep it here close to the client.